framework/include/multiapps/MultiApp.h

```diff
@@ -276,6 +276,15 @@ class MultiApp : public MooseObject, public SetupInterface, public Restartable
    */
   bool isRootProcessor() { return _my_rank == 0; }
 
+  /**
+   * Reserve the solution from the previous simulation,
+   *  and it is used as an initial guess for the next run
+   */
+  void keepSolutionDuringRestore(bool keep_solution_during_restore)
```

Comment:
This method should not be public. Also, there should be a warning or at least information when the value passed in directly conflicts with an explicit value set in the input file.